purchase_packaging_level_qty/views/purchase_order_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="purchase_order_form_inerit" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']//tree//field[@name='price_unit']"
+                position="before"
+            >
+                <field name="transport_qty" optional="hide" />
+            </xpath>
+            <xpath expr="//field[@name='tax_totals']" position="after">
+                <label
+                    for="total_transport_qty"
+                    attrs="{'invisible': [('total_transport_qty', '=', False)]}"
+                />
+                <div class="text-nowrap">
+                    <field
+                        name="total_transport_qty"
+                        class="oe_inline"
+                        attrs="{'invisible': [('total_transport_qty', '=', False)]}"
+                    />
```

Comment:
Add the new `transport packaging` field to see `30.5 Pallet` instead of just `30.5`